### ISSUE TITLE
Error M7701-1003

### ISSUE THREAD

**Issue by Alteux** [nirewen/discord-netflix#82]
Netflix Error M7701-1003
If you experience the error code M7701-1003 on your computer, often accompanied by the following message:

Whoops, something went wrong...
Missing Component
We cannot find all the required components to play Netflix on this device. Please visit chrome://components, locate the WidevineCdm component, and click the "Check for update" button.
It typically points to a component on your browser that needs to be updated or that is being blocked by a security setting or program. Follow the troubleshooting steps below to resolve the issue.

Using MacOS Catalina

**Comment by MJ94** [comment]
It looks like this may be a duplicate of issue #63.
